tests/samples/cmake-files/FindPythonExtensions.cmake

```diff
@@ -222,37 +222,48 @@
 find_package( PythonInterp REQUIRED )
 find_package( PythonLibs REQUIRED )
 
-set(_command "")
-set(_command "${_command}import distutils.sysconfig\n")
-set(_command "${_command}import itertools\n")
-set(_command "${_command}import os\n")
-set(_command "${_command}import os.path\n")
-set(_command "${_command}import site\n")
-set(_command "${_command}import sys\n")
-
-set(_command "${_command}result = None\n")
-set(_command "${_command}rel_result = None\n")
-set(_command "${_command}candidates = itertools.chain(\n")
-set(_command "${_command}  (distutils.sysconfig.get_python_lib(),),\n")
-set(_command "${_command}  site.getsitepackages(),\n")
-set(_command "${_command}  (site.getusersitepackages,)\n")
-set(_command "${_command})\n")
-
-set(_command "${_command}for candidate in candidates:\n")
-set(_command "${_command}    rel_candidate = os.path.relpath(\n")
-set(_command "${_command}      candidate, sys.prefix)\n")
-set(_command "${_command}    if not rel_candidate.startswith('..'):\n")
-set(_command "${_command}        result = candidate\n")
-set(_command "${_command}        rel_result = rel_candidate\n")
-set(_command "${_command}        break\n")
-
-set(_command "${_command}sys.stdout.write(\";\".join((\n")
-set(_command "${_command}    os.sep,\n")
-set(_command "${_command}    os.pathsep,\n")
-set(_command "${_command}    sys.prefix,\n")
-set(_command "${_command}    result,\n")
-set(_command "${_command}    rel_result,\n")
-set(_command "${_command})))\n")
+set(_command "
```

Comment:
It's not that important exactly where it lives, since we should be planning to send it upstream when its ready.  I chose `tests/samples/cmake-files` to keep them near the sample projects that would use them.